management/management.py

```diff
@@ -176,6 +176,13 @@ def restart_optimizations(optimizations, client):
             client.modify_tasks(operation='restart', base_result=opt.id)
 
 
+def regenerate_optimizations(optimizations, client):
+    for opt in optimizations:
+        if opt.status == 'INCOMPLETE' and (opt.trajectory is not None):
+            print(opt)
+            client.modify_tasks(operation='regenerate', base_result=opt.id)
```

Comment:
Given that we don't know how well this will work, it might be helpful to print some context with this optimization, like "this optimization ID ### was regenerated"

Reply:
Will add additional context, good call!

Reply:
Done! Thanks Trevor!